docs/source/conf.py

```diff
@@ -11,10 +11,9 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
 import os
 import sys
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath(os.path.join(__file__, '..', '..')))
```

Comment:
Interesting approach.  I did not realize you could do .. after a file.

Reply:
Well, I guess you can, although this produces the wrong result!

```
In [73]: os.path.abspath(os.path.join("/Users/ttung/microscopy/starfish/docs/source/conf.py", "..", ".."))
Out[73]: '/Users/ttung/microscopy/starfish/docs'
```

Reply:
🤦‍♂️ 
Fixed.